core/pkg/hostsensorutils/hostsensor.yaml

```diff
@@ -52,12 +53,19 @@ spec:
         volumeMounts:
         - mountPath: /host_fs
           name: host-filesystem
-        readinessProbe:
+        startupProbe:
           httpGet:
-            path: /kernelVersion
+            path: /readyz
             port: 7888
-          initialDelaySeconds: 1
-          periodSeconds: 1
+          initialDelaySeconds: 10
+          failureThreshold: 10
+          periodSeconds: 10
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 7888
+          initialDelaySeconds: 5
```

Comment:
don't need initial delay, since the startupProbe protects it

Reply:
ok